api/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM rust:1.63.0
+
+WORKDIR /usr/src/app/calandar
+
+EXPOSE 8000
+
+RUN cargo install cargo-watch
+RUN cargo install sqlx-cli
+
+COPY . .
```

Comment:
nb that cargo's cache is in your home directory. If recompiles are taking a toll on your life, use cargo locally.

Reply:
OK, I'll investigate how to set the cache directory.

Reply:
I was able to change the `CARGO_HOME` directory, but it didn't work to improve recompilation time under most conditions. 

These docker image builds need an overhaul soon anyway, but that shouldn't block functions right now - so long as the inner loop is fast, which it is.

I'll come back to this down the line. Thanks for the suggestion